drivers/hmis/spec/models/hmis/hud/enrollment_spec.rb

```diff
@@ -227,4 +227,122 @@
       end
     end
   end
+
+  describe 'occurrence point form instances' do
+    let(:role) { :OCCURRENCE_POINT }
+    let!(:project) { create(:hmis_hud_project, data_source: ds1) }
+    let!(:hoh_enrollment) { create(:hmis_hud_enrollment, project: project, data_source: ds1, household_id: 'household1', relationship_to_hoh: 1) }
+    let!(:spouse_enrollment) { create(:hmis_hud_enrollment, project: project, data_source: ds1, household_id: 'household1', relationship_to_hoh: 3) }
+
+    context 'no relevant instance exists' do
+      it 'does not return the form when no instance exists' do
+        expect(hoh_enrollment.occurrence_point_forms).to be_empty
+      end
+
+      it 'does not return the form when a draft instance exists' do
+        definition = create(:hmis_form_definition, role: role, status: :draft)
+        create(:hmis_form_instance, role: role, entity: project, active: true, definition: definition)
+        expect(hoh_enrollment.occurrence_point_forms).to be_empty
+      end
+
+      it 'does not return when an irrelevant instance exists' do
+        create(:hmis_form_instance, role: role, entity: p1, active: true) # applies to a different project
+        expect(hoh_enrollment.occurrence_point_forms).to be_empty
+      end
+
+      it 'does not return when an inactive instance exists' do
+        create(:hmis_form_instance, role: role, entity: p1, active: false)
+        expect(hoh_enrollment.occurrence_point_forms).to be_empty
+      end
+    end
+
+    context 'when an instance exists relevant to this project' do
+      let!(:definition_json) do
+        {
+          'item': [
+            {
+              'text': 'Move-in Date',
+              'type': 'DATE',
+              'link_id': 'moveInDate',
+              'mapping': {
+                'field_name': 'moveInDate',
+                'record_type': 'ENROLLMENT',
+              },
+            },
+            {
+              'text': 'Move in address',
+              'type': 'OBJECT',
+              'link_id': 'moveInAddresses',
+              'mapping': {
+                'field_name': 'moveInAddresses',
+                'record_type': 'ENROLLMENT',
+              },
+              'repeats': false,
+              'component': 'ADDRESS',
+            },
+          ],
+        }
+      end
+      let!(:definition) { create :hmis_form_definition, role: role, definition: definition_json }
+      let!(:instance) { create(:hmis_form_instance, role: role, entity: project, active: true, definition: definition) }
```

Comment:
add test case for what happens for an enrollment in a different project that has a MoveInDate